lib/ui/common/widgets/ActionSheet.dart

```diff
@@ -0,0 +1,193 @@
+import 'package:farmsmart_flutter/ui/common/ActionSheetLargeRoundedButtonStyle.dart';
+import 'package:farmsmart_flutter/ui/common/ListDivider.dart';
+import 'package:farmsmart_flutter/ui/common/headerAndFooterListView.dart';
+import 'package:farmsmart_flutter/ui/common/roundedButton.dart';
+import 'package:flutter/material.dart';
+
+class ActionListItemViewModel {
+  String title;
+  String icon;
+  String checkBoxIcon;
+  bool isHighlighted;
```

Comment:
in iOS, the red action items are named "destructive". it's a warning to the user that the action will delete, or remove something. I think this should be named "isDestructive"

Reply:
Done